src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -155,4 +159,22 @@ public void shutdownCompleted(ShutdownSignalException cause) {
     private Channel giveMeRandomChannel() {
         return rabbitChannels.get(random.nextInt(rabbitChannels.size()));
     }
+    
+    public String getEventId(JsonElement json){
```

Comment:
This functionality does not scale. Consider https://github.com/Ericsson/eiffel-remrem-generate/pull/33 where we add dynamic support of libraries. Any library could be used. Must be solved with a call to the library via the shared interface.